WordPressAuthenticator/Signin/LoginViewController.swift

```diff
@@ -299,11 +299,15 @@ extension LoginViewController {
             return
         }
 
+        // FIXME: Tests this
         let appleConnectParameters: [String: AnyObject]? = {
-            if let appleUser = loginFields.meta.appleUser {
-                return AccountServiceRemoteREST.appleSignInParameters(email: appleUser.email, fullName: appleUser.fullName)
-            }
-            return nil
+            guard let user = loginFields.meta.socialUser else { return nil }
+            guard case .apple = user.service else { return nil }
```

Comment:
Nitpick: merging these two in to one guard statement?

Reply:
Nice. Addressed in 7d91349a